seqr_scripts/import_vcf.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.utils.filtering import subset_samples_and_variants
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("subset")
+logger.setLevel(logging.INFO)
+
+
+def main(args):
+
+    hl.init(
+        default_reference=args.import_build,
+        log="/import_vcf.log",
+        tmp_dir="hdfs:///import_vcf.tmp/",
+    )
+
+    logger.info("Importing filters info...")
+    filters_ht = hl.import_vcf(
+        args.part_two_path,
+        reference_genome=args.import_build,
+        force_bgz=True,
+        find_replace=("nul", "."),
+    ).rows()
+    logger.info("Importing VCF...")
+    # Note: always assumes file is bgzipped
```

Comment:
```suggestion
    # NOTE: always assumes file is bgzipped
```
optional, I think most of the comments that start with note have "NOTE:" in the gnomad repo